apps/shell/src/app/modules/settings/pages/settings-general/github-conenction/github-connection.tsx

```diff
@@ -18,6 +19,8 @@ export function GithubConnection() {
     (actions) => actions.settings.disconnectGitHub,
   );
 
+  const formState = nanoid();
```

Comment:
You'll want to memo this so it doesn't get scrambled on repaint.

```suggestion
  const formState = useMemo(() => nanoid(), []);
```